styles/twitch.scss

```diff
@@ -0,0 +1,283 @@
+$bright-white: #fff;
+$brand-white: #faf9fa;
+$light-purple: #cbc8d0;
+$brand-purple: #17141f;
+$button-purple: #6441a4;
+
+$gutter: 1rem;
+
+* {
+  // border: 1px solid pink;
+}
+
+ul {
+  list-style: none;
+  margin-left: 0;
+}
+
+.tw-app {
+  height: 100vh;
+  width: 100%;
+  overflow-y: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+.favicon {
+  height: 16px;
+  width: auto;
+}
+
+.logo {
+  height: 2rem;
+  width: auto;
+  margin: 1rem auto;
+  display: block;
+}
+
+.tw-app__left {
+  display: flex;
+  justify-content: center;
+  // align-items: stretch;
+  background: $brand-purple;
+  width: 28%;
+  height: 100%;
+  overflow: auto;
+
+  padding: $gutter;
+}
+
+.InputAddOn {
+  display: flex;
+  flex-direction: row;
+  margin: auto;
+  color: $bright-white;
+  font-family: Helvetica Neue,Helvetica,sans-serif;
+}
+
+.InputAddOn-item {
+  margin-right: 0.3rem;
+}
+
+.nav-list {
+  color: $light-purple;
+  margin: 0.5rem auto;
+  font-family: Helvetica Neue, Helvetica, sans-serif;
+}
+
+.nav-list__item {
+  padding: 0.5rem;
+  font-family: Helvetica Neue,Helvetica,sans-serif;
+}
+
+.channels-list {
+  margin: 1rem auto;
+}
+
+.channels {
+  color: $light-purple;
+  text-transform: uppercase;
+  font-family: Helvetica Neue,Helvetica,sans-serif;
+  font-size: 0.9rem;
+}
+
+.channels-list__item {
+  display: flex;
+}
+
+.image-wrap {
+
+}
+
+.text-wrap {
+  margin-left: 0.5rem;
+}
+
+.list__item-title {
+  color: $light-purple;
+  font-size: 0.9rem;
+}
+
+.item-title__desc {
+  color: gray;
+  display: block;
+  font-size: 0.7rem;
+}
+
+.other-list__container {
+  margin: 1rem auto;
+}
+
+.other-list {
+  display: flex;
+  justify-content: space-between;
+  margin-top: -1rem;
+}
+
+.more-twitch__item {
+  color: $light-purple;
+  font-size: 0.7rem;
+  padding: 0.2rem 0 0.2rem 0;
+}
+
+.btn-container {
+  text-align: center;
+}
+
+.nav-btn {
+  background: $button-purple;
+  color: $bright-white;
+  border: none;
+  padding: 0.5rem 1rem;
+  font-size: 0.8rem;
+}
+
+.tw-app__center {
+  display: block;
+  position: relative;
+  background: $bright-white;
+  width: 80%;
+  overflow: auto;
+  flex-direction: column;
+  margin: 2rem;
```

Comment:
Using `margin` here makes the scrollbars a bit awkward, if you change this to `padding` it makes things 👍